### PR TITLE
adds orangehrm chart

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

File: stable/orangehrm/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dependencies:
+- name: mariadb
+  version: 0.5.4
```
Comment (viglesiasce) [inline stable/orangehrm/requirements.yaml]: Bump this to 0.5.6 which has the fix for imagePullPolicy on Kubernetes 1.5.1

**Review by viglesiasce — CHANGES_REQUESTED** [review]

**Comment by sameersbn** [comment]
done

**Review by viglesiasce — APPROVED** [review]

**Comment by viglesiasce** [comment]
@k8s-bot e2e test this

**Pull request merged**